.gitignore

```diff
@@ -4,3 +4,5 @@
 .DS_Store
 docs/build/
 docs/site/
+/.benchmarkci
+/benchmark/*.json
```

Comment:
Are these files appearing?  How are they  generated?

Reply:
Locally these files appear for me. 
They are generated while saving the results of benchmark to the file look [here](https://github.com/tkf/BenchmarkCI.jl/blob/8ccfbd704a3454cfc04867819f3580bd6a748eef/src/BenchmarkCI.jl#L75)